analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -116,6 +129,14 @@ abstract class PackageManager(
                             return FileVisitResult.SKIP_SUBTREE
                         }
 
+                        if (excludes.isPathExcluded(rootPath.relativize(dir).pathString)) {
+                            logger.info {
+                                "Not analyzing directory '$dir' as it is matched by a path exclude."
```

Comment:
optional: Maybe `PathExclude` is not needed implementation detail in "as it is matched by a path exclude.". Would it make sense to say "Not analyze directory '$dir' as it is excluded."?